### PR TITLE
Fix: Fixed issue with tab icon not loading properly

### LINKED ISSUE #9929: Bug: Icons are no longer shown on the tab control
**Requirements**
- Display the icon of current directory on the tab control

### PR DESCRIPTION
**Pull request by chingucoding**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9929

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 1]
```diff
@@ -234,7 +234,7 @@
 												Margin="{ThemeResource TabViewItemHeaderIconMargin}">
 												<ContentControl
 													x:Name="IconControl"
-													Content="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TabViewTemplateSettings.IconElement}"
+                                                    Content="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TabViewTemplateSettings.IconElement, Mode=OneWay}"
```
Comment (yaira2) [inline src/Files.App/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:237]: Spacing is messed up, can you make sure it's set to tabs?

**Review by yaira2 — APPROVED** [review]
Tab control is pretty again!
LGTM

**Comment by yaira2** [comment]
@onein528 fyi, this might be useful for Fluent Hub.

**Comment by hecksmosis** [comment]
Should FilesLauncher be comitted?

**Comment by yaira2** [comment]
We should add to gitignore for the future but it doesn't really matter for this PR.

**Comment by hecksmosis** [comment]
Should I add it?

**Comment by yaira2** [comment]
Sure

**Comment by onein528** [comment]
Thanks!

**Pull request merged**